spec/requests/v1.0/requests_spec.rb

```diff
@@ -14,7 +14,7 @@
   let!(:requests_with_same_state) { create_list(:request, 2, :state => 'notified', :workflow_id => workflow.id, :tenant_id => tenant.id) }
   let!(:requests_with_same_decision) { create_list(:request, 2, :decision => 'approved', :workflow_id => workflow.id, :tenant_id => tenant.id) }
```

Comment:
I noticed `spec/requests/` uses a `v1.0` directory structure and `spec/controllers/` uses a `v1` directory structure. Is there a reason for the difference?